release/cut-official-release.sh

```diff
@@ -0,0 +1,338 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script automates the release processes for all pre-releases and official
+# releases for Kubernetes.  See docs/devel/releasing.md for more info.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Sets DIR, INSTRUCTIONS
+function main() {
+  # Parse arguments
+  if [[ "$#" -ne 2 ]]; then
+    usage
+    exit 1
+  fi
+  local -r new_version=${1-}
+  # TODO(ihmccreery) Stop calling it githash; it's not a githash.
```

Comment:
No, the change would have to be cherrypicked, so it would need to be a separate PR.